<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumpDataDetailedInfoWindow</class>
 <widget class="QWidget" name="JumpDataDetailedInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>508</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Informacje o skoku</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_jumperInfoTitle">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_nameSurnameAndFlag">
         <item>
          <widget class="QLabel" name="label_nameAndSurname">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Kamil Stoch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_flag">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>IMG</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_mainInformations">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_distanceTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Odległość</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_distance">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(6, 75, 150);</string>
         </property>
         <property name="text">
          <string>123m</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_pointsTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Punkty</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_points">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(168, 40, 0);</string>
         </property>
         <property name="text">
          <string>119.8pkt</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_gateTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Belka startowa</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_gate">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_gatePointsTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Punkty za belkę</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_gatePoints">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_averageWindTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Uśredniony wiatr</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_averageWind">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_windCompensationTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Punkty za wiatr</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_windPoints">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_totalCompensationTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Łączna rekompensata</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_totalCompensation">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_judgesTItle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Noty sędziowskie</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_judges">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(168, 105, 0);</string>
         </property>
         <property name="text">
          <string>| 17.5 | 18 | 18 | 17 | 17.5 |</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_judgesPointsTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Punkty za styl</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_judgesPoints">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(80, 58, 10);</string>
         </property>
         <property name="text">
          <string>53pkt</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_landingTypeTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Rodzaj lądowania</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_landingType">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Telemark</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_positionInRoundTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Miejsce w serii</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label_positionInRound">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_positionInGroupTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Miejsce w grupie</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_positionInGroup">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color :black;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_positionInCompetitionTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Miejsce w konkursie</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="label_positionInCompetition">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_beats95HSPercents">
       <property name="font">
        <font>
         <family>Quicksand Medium</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(49, 76, 44);</string>
       </property>
       <property name="text">
        <string>Zawodnik osiągnął 95% punktu HS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_DSQ">
       <property name="font">
        <font>
         <family>Quicksand</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(116, 0, 2);</string>
       </property>
       <property name="text">
        <string>DYSKWALIFIKACJA</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_windsInfo">
       <property name="spacing">
        <number>2</number>
       </property>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Quicksand Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(111, 172, 94);
color: rgb(40, 40, 40);
border-radius: 10px;
}
QPushButton:hover{
background-color: rgb(81, 142, 64);
}</string>
       </property>
       <property name="text">
        <string>Wyślij webhook-a</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_others">
     <item>
      <layout class="QFormLayout" name="formLayout_simulationRating">
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_takeoffRatingTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Ocena wybicia</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_randomTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Losowość</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_flightRatingTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Ocena lotu</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_takeoffRating">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(80, 17, 17);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_flightRating">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(80, 17, 17);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_random">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(80, 17, 17);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_snowRainTitle">
         <property name="font">
          <font>
           <family>Quicksand</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>color: rgb(20, 20, 20);</string>
         </property>
         <property name="text">
          <string>Śnieg/Deszcz</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_snowRain">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(80, 17, 17);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
